ios/PurchasesHybridCommon/PurchasesHybridCommon/ErrorContainer.swift

```diff
@@ -27,6 +27,14 @@ import RevenueCat
 
         info["underlyingErrorMessage"] = underlyingErrorMessage ?? ""
 
+        if let storeKitError = ErrorContainer.findStoreKitErrorCodeIfAny(nsError) {
+            info["storeError"] = [
```

Comment:
Note that this is not used by some hybrids (like RN, which uses the original error). But this will include this information for those hybrids that do.

Reply:
That's correct... We would need to figure out a way since in RN, we just pick the original error, but yeah, I think that makes a lot of sense 👍